src/mutations/InstrumentMutations.ts

```diff
@@ -139,14 +142,31 @@ export default class InstrumentMutations {
     );
 
     if (!allProposalsAreOnSameCallAsInstrument) {
+      logger.logError(
+        'Can not assign the proposal to the instrument because proposals call has no such intrument',
```

Comment:
'instrument' typo, plus optional tweaks
```suggestion
        'Cannot assign the proposal to the instrument because the proposals call has no such instrument',
```